staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -54,6 +55,7 @@ type RequestScope struct {
 	Defaulter       runtime.ObjectDefaulter
 	Typer           runtime.ObjectTyper
 	UnsafeConvertor runtime.ObjectConvertor
+	Authorizer      authorizer.Authorizer
```

Comment:
Surprised to see this in both group info and here. Maybe it will make sense after I read more.

Reply:
I guess it is copied here from the group info struct for each request? It seems a little strange to me to copy this into the APIGroupVersion struct at all--it should be constant for the whole apiserver, not vary by endpoint, right?

Reply:
Yeah, It's the same for all the endpoints, but I thought this was the only way to expose it to the installer and then to the individual handlers